perfkitbenchmarker/benchmarks/aerospike_benchmark.py

```diff
@@ -48,23 +49,31 @@
                      'The percent of operations which are reads.',
                      lower_bound=0, upper_bound=100)
 
-BENCHMARK_INFO = {'name': 'aerospike',
-                  'description': 'Runs Aerospike',
-                  'num_machines': 2}
+BENCHMARK_NAME = 'aerospike'
+BENCHMARK_CONFIG = """
+aerospike:
+  description: Runs Aerospike.
+  vm_groups:
+    default:
+      vm_spec: *default_single_core
+      disk_spec: *default_500_gb
+      vm_count: 2
```

Comment:
Could you add TODOs and/or open an issue where appropriate to modify the specs for client/server benchmarks like this? Disks should only be needed on the server.